### PR TITLE
feat/default-flow-creation-template

### LINKED ISSUE #1271: Provide more comprehensive template for flow creation init
### Feature description

It would be great to have this template instead of the existing one since flows need at least one task to be valid
```
id: ""
namespace: ""
tasks:
  - id: ""
    type: ""
```

### PR DESCRIPTION
**Pull request by brian-mulier-p**
closes #1271

### CODE REVIEW

**Review by Skraye — CHANGES_REQUESTED** [review]

File: ui/src/components/graph/nodes/Edge.vue [hunk 1]
```diff
@@ -132,6 +132,7 @@
 
         const leftNodeIsTask = YamlUtils.extractTask(props.yamlSource, props.id.split("|")[0]) !== undefined;
         if (leftNodeIsTask) {
+            [..."hello"].
```
[the remark this answers is not in the document]
Comment (brian-mulier-p) [reply ui/src/components/graph/nodes/Edge.vue:135]: Oups, I was testing something, good spot :+1:

[the remark this answers is not in the document]
Comment (Skraye) [reply ui/src/components/graph/nodes/Edge.vue:135]: I don't have the issue; maybe you have some dependencies that are not up to date?

File: ui/src/components/inputs/EditorView.vue [hunk 2]
```diff
@@ -163,20 +163,13 @@
...
-            const validationErrors = validation[0].constraints;
-            if (validationErrors) {
-                singleErrorToast(t("cannot create topology"), t("invalid flow"), validationErrors);
-            }else {
-                generateGraph();
-            }
-        } else {
+        const validation = await store.dispatch("flow/validateFlow", {flow: flowYaml.value});
+        const validationErrors = validation[0].constraints;
+        if (validationErrors) {
```
[the remark this answers is not in the document]
Comment (brian-mulier-p) [reply ui/src/components/inputs/EditorView.vue:170]: I feel like we weren't handling that case unless in creation mode. Is that intended ? Should we create another issue ? We may need to add another validate then or should we simply call fetchGraph and handle the error case ?

File: ui/src/components/graph/nodes/Edge.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (brian-mulier-p) [reply ui/src/components/graph/nodes/Edge.vue:135]: I just npm installed and still have the issue, looks strange :thinking:

[the remark this answers is not in the document]
Comment (brian-mulier-p) [reply ui/src/components/graph/nodes/Edge.vue]: I guess I will try to live along this compiler error

**Pull request merged**